CHANGELOG.md

```diff
@@ -41,6 +41,8 @@
 #### 🚀 Enhancement
 
 - feat(design tokens): style dictionary replaces css-in-js from virtru-design-params [#19](https://github.com/avkvirtru/virtuoso-design-system/pull/19) ([@avkvirtru](https://github.com/avkvirtru))
+- feat: add dark theme to header [#47](https://github.com/avkvirtru/virtuoso-design-system/pull/47) ([@aleonov-virtru](https://github.com/aleonov-virtru))
+- feat: add title icon to header [#47](https://github.com/avkvirtru/virtuoso-design-system/pull/47) ([@aleonov-virtru](https://github.com/aleonov-virtru))
 
```

Comment:
@aleonov-virtru The changelog is autogenerated from the commit messages. In this case we'd want to rebase your messages to include the data you provided.

Ideally with [conventional-commits](conventionalcommits.org/en/v1.0.0/) you only have one commit message per PR, but multiple is fine with me. Just keep in mind that if you want to commit two messages like above you'd need to commit the work while you're developing it.